lib/phlex/html.rb

```diff
@@ -136,17 +136,29 @@ def call(...)
 				new(...).call
 			end
 			alias_method :render, :call
+
+			def new(*args, &block)
+				if block
+					object = super(*args, &nil)
```

Comment:
We don't pass the block up to `super` because we don't want it to be handled by a users initialiser.